File: Travelo.Model/PaymentMethod.cs
using System;
namespace Travelo.Model
{
	public partial class PaymentMethod
	{
        public int Id { get; set; }
        public string CardNumber { get; set; }
        public string HolderName { get; set; }
        public string Address { get; set; }
        public string CVV { get; set; }
        public string ExpDate { get; set; }
        public string CardType { get; set; }
        public string LastDigits { get; set; }
    }
}

File: travelo_mobile/lib/model/paymentMethod.dart
import 'package:json_annotation/json_annotation.dart';

part 'paymentMethod.g.dart';

@JsonSerializable()
class PaymentMethod {
  int? id;
  String? cardNumber;
  String? holderName;
  String? address;
  String? cvv;
  String? expDate;
  bool? primary;
  String? cardType;
  String? lastDigits;
  int? userId;

  PaymentMethod();

  factory PaymentMethod.fromJson(Map<String, dynamic> json) =>
      _$PaymentMethodFromJson(json);
}
